src/babel.ts

```diff
@@ -44,6 +44,19 @@ export default function transform(opts: TransformOptions): TRANSFORM_RESULT {
     _opts.plugins.push(require("@babel/plugin-syntax-import-assertions"));
   }
 
+  if (opts.jsx) {
+    if (!_opts.presets) {
+      _opts.presets = [];
+    }
+
+    _opts.presets.push([
+      require("@babel/preset-react"),
```

Comment:
I think we should base on `synctax-jsx` directly for basic/universal support https://babeljs.io/docs/babel-plugin-syntax-jsx

Reply:
I use `@babel/preset-react` because `@babel/plugin-syntax-jsx` can only support babel to parse this syntax, but cannot transform to code to `h` function or `React.createElement`. Or we can use both  `@babel/preset-react` and `@babel/plugin-transform-react-jsx` to parse it and transform to `h` function (or `React.createElement`). `@babel/preset-react` just do it inside.

Reply:
I use plugins to replace presets.

Reply:
Thanks for change. I will try to locally test but as explained in #138, my main idea to have minimal syntax changes and instead _inject_ the react (or any other JSX renderer) render function in instead of tightly coupling with react (although rest of the ecosystem for many of historical reasons are currently are)

Reply:
I don't know if I understand correctly. It seems that we didn't tightly couple with react. Installing `react` as devDependencies is to solve the test error when running jsx files. And `@babel/plugin-transform-react-jsx` can transform code from `<div>foo</div>` to `React.createElement('div', null, 'foo')` (or `h('div', null, 'foo')` depends on configuration. That's all we do. And the react (or other JSX renderer) is injected by the users.